aiohttp/helpers.py

```diff
@@ -240,18 +240,17 @@ class AccessLogger:
     Format:
         %%  The percent sign
         %a  Remote IP-address (IP-address of proxy if using reverse proxy)
-        %t  Time the request was received
+        %t  Request's recieving timestmap
```

Comment:
It is more like response time. Also it is not quiet timestamp, it is time in common log format. Actually I just copied this descriptions from apache's docs.